src/lib/generate-og.js

```diff
@@ -10,9 +9,7 @@ const OpenGraph = require("../components/open-graph");
 function render(ui) {
   return new Promise(resolve => {
     let html = "";
-    const stream = renderToStaticNodeStream(ui).pipe(
-      renderStylesToNodeStream()
```

Comment:
Are you sure this is not required?

Reply:
@sergiodxa  I'm getting an error by running `npx @contentz/social home` for example

```
Cannot find module 'emotion'
Require stack:
- /Users/cristianbgp/.npm/_npx/65670/lib/node_modules/@contentz/social/node_modules/emotion-server/dist/emotion-server.cjs.dev.js
```
So I was testing removing `emotion-server` and it still works by running locally this project `node cli.js home`

Can you help me testing this? [I don't have idea what removing this line implicates](https://i.imgur.com/BjymSUX.gif)